server/integration_tests/test_data/detection_api_bugs/chart_config.json

```diff
@@ -15,25 +15,7 @@
   "comparisonEntities": [],
   "comparisonVariables": [
     "dc/topic/HousesByHomeValue",
-    "Count_HousingUnit_HomeValue35000To39999USDollar",
-    "Count_HousingUnit_HomeValue50000To59999USDollar",
-    "Count_HousingUnit_HomeValue175000To199999USDollar",
-    "Count_HousingUnit_HomeValue60000To69999USDollar",
-    "Count_HousingUnit_HomeValue200000To249999USDollar",
-    "Count_HousingUnit_HomeValue25000To29999USDollar",
-    "Count_HousingUnit_HomeValue250000To299999USDollar",
-    "Count_HousingUnit_HomeValue70000To79999USDollar",
-    "Count_HousingUnit_HomeValue40000To49999USDollar",
-    "dc/topic/HousesByOwnershipCost",
-    "Count_HousingUnit_HomeValue30000To34999USDollar",
-    "Count_HousingUnit_HomeValue20000To24999USDollar",
-    "Count_HousingUnit_HomeValue15000To19999USDollar",
-    "Count_HousingUnit_HomeValue80000To89999USDollar",
-    "Count_HousingUnit_HomeValue125000To149999USDollar",
-    "Count_HousingUnit_HomeValue90000To99999USDollar",
-    "Count_HousingUnit_HomeValue150000To174999USDollar",
-    "Count_HousingUnit_HomeValue2000000OrMoreUSDollar",
-    "Count_HousingUnit_HomeValueUpto10000USDollar"
+    "dc/topic/HousesByOwnershipCost"
```

Comment:
diff is OK, but just need to check why the topic order flipped compared to dev.

Reply:
From debug page, the variables look very different from the golden here. 

Query "What is the relationship between housing size and home prices in California". 

should the chart_config.json be exact same as what shows up on the page?